tests/test_run_scripts/acceptance_tests.sh

```diff
@@ -12,8 +12,9 @@ declare -a ARGS
 if [ $# -eq 0 ]; then
   ARGS[0]='windows-2012r2-64a'
   ARGS[1]='1.2.2'
-elif [ $# -ne 2 ]; then
-  echo 'USAGE acceptance_tests.sh <CONFIG> <PUPPET_AGENT_VERSION>'
+  ARGS[2]='true'
```

Comment:
Passing true at the command line like this is a little weird... but I suspect you'll be the only one using this script @cowofevil